.github/workflows/tests.yml

```diff
@@ -19,15 +19,25 @@ jobs:
       - name: install cookiecutter
         run: python -m pip install cookiecutter
 
+        # See this PR for more details https://github.com/actions/checkout/pull/1184#issue-1595060720
+      - name: setup GitHub bot git committer
+        run: |
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
```

Comment:
Just to make sure I understand -- we need the `git config` setup here to make sure the subsequent `git commit` in each test works, right?

Reply:
Correct! I think this is necessary unless you know of a better way to commit without setting up the username and email.

Reply:
Nope, none that I can think of. One suggestion: since these are getting disposed anyways, let's the username/email `fake@example.com` to avoid any confusion about what they signify 🙂

Reply:
Done! Also replaced some text to make things clearer.